packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -18,26 +18,11 @@
         data-testid="sharingInfo"
         class="oc-flex oc-flex-middle oc-my-m"
       >
-        <oc-button
-          v-if="hasPeopleShares"
-          v-oc-tooltip="peopleSharesLabel"
-          appearance="raw"
-          class="oc-mr-xs"
-          :aria-label="peopleSharesLabel"
-          @click="expandPeoplesPanel"
-        >
-          <oc-icon name="group" />
-        </oc-button>
-        <oc-button
-          v-if="hasLinkShares"
-          v-oc-tooltip="linkSharesLabel"
-          appearance="raw"
-          class="oc-mr-xs"
-          :aria-label="linkSharesLabel"
-          @click="expandLinksPanel"
-        >
-          <oc-icon name="link" />
-        </oc-button>
+        <oc-status-indicators
+          v-if="indicators && indicators.length > 0"
```

Comment:
```suggestion
          v-if="indicators.length"
```
is sufficient. the computed prop always returns an array.